### DISCUSSION TITLE
The behaviour when a key is pressed and held down

### DISCUSSION THREAD

**Discussion by chriscroome** [kovidgoyal/kitty#5307] [Q&A] [answered]
Hi, I'm new to Kitty, after many years of using xterms, I'm used to the behaviour of a key that is held down, for example the left, right, up, down arrows or backspace, to result in the key being sent multiple times to the application.

With Kitty it appears that by sometimes pressing a key down and holding it down results in the same behaviour as pressing it once and sometimes the key is repeated, I have both behaviours happening in different Kitty windows, which makes me think that I must have some legacy configuration in my home directory causing this perhaps?

I have had a look for an option to configure this behaviour in Kitty but couldn't find it, perhaps I'm searching for the wrong thing? Any idea what might be going on here (chances are that it is a user error...)?

Thanks.

**Comment by page-down** [comment, marked as answer]
I noticed the following issue that has been fixed but not officially released.

https://github.com/kovidgoyal/kitty/issues/5188

If you are using wayland, maybe you can try nightly release and see if the problem still exists.

You can also run `kitty --debug-input` to troubleshoot the problem.

**Reply by chriscroome** [reply]
Thanks! Sorry for not finding that issue, I'm also running Sway with Wayland, but on Debian Bookworm rather than Arch, I have [installed the latest Kitty using Ansible](https://git.coop/webarch/kitty) -- I'll now add support for installing [Nightly builds](https://github.com/kovidgoyal/kitty/releases/tag/nightly) in addition to [Latest](https://github.com/kovidgoyal/kitty/releases/latest) to the role... :+1:
